.github/workflows/test.yml

```diff
@@ -109,3 +109,51 @@ jobs:
         run: |
           echo "Total coverage ${{ steps.jacoco.outputs.coverage-overall }}"
           echo "Changed Files coverage ${{ steps.jacoco.outputs.coverage-changed-files }}"
+
+  test-amd64-arch:
+    needs: build
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        architecture: [linux/amd64]
+        java-version: [ 8, 11, 17 ]
+        test-mode: [ embedded, deploy ]
+    env:
+      DOCKER_RUN_NAME: hive-amd64
+      PLATFORM: ${{ matrix.architecture }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java-version }}
+          distribution: 'temurin'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Package Gravitino
+        run: |
+          ./gradlew build -x test -PjdkVersion=${{ matrix.java-version }}
```

Comment:
Is this step still required? As we've already build the project above.

Reply:
If the job name is 'Package Gravitino' and it is indeed a packaging task, the `build` task can be skipped and use `compileDistribution` directly.

Reply:
fixed.

Reply:
I tried. But It will fail.

Reply:
I execute `./gradlew compileDistribution -x test` locally with executing `./gradlew build`, and it seems no error exists

Reply:
It doesn't work. You can see https://github.com/datastrato/gravitino/actions/runs/7536000721

Reply:
I wander why `./gradlew compileDistribution -x test` will trigger integration test...

Reply:
> I wander why `./gradlew compileDistribution -x test` will trigger integration test...

We have next steps. The step which you mentioned doesn't trigger.

Reply:
We have already built the Gravitino in line 85, I was thinking why can't we reuse the built package?

Reply:
They're different jobs.  Different jobs may be scheduled to different machines.